pkg/utils/images/images.go

```diff
@@ -63,6 +63,8 @@ const (
 	ImageNameHyperkube = "hyperkube"
 	// ImageNameIngressDefaultBackend is a constant for an image in the image vector with name 'ingress-default-backend'.
 	ImageNameIngressDefaultBackend = "ingress-default-backend"
+	// ImageNameInitLargeDir is a constant for an image in the image vector with name 'init-large-dir'.
+	ImageNameInitLargeDir = "init-large-dir"
```

Comment:
Similarly, I would propose to rename `ImageNameInitLargeDir` to a more specific name.
WDYT?